src/containers/App.tsx

```diff
@@ -13,33 +18,65 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     backgroundColor: '#F5FCFF',
   },
+  button: {
+    color: '#841584',
+  },
 });
 
 interface IProps {
+  addCounter: () => void;
   decreaseCount: (counterId: number) => void;
   increaseCount: (counterId: number) => void;
+  removeCounter: (counterId: number) => void;
   counters: ICounter[];
 }
 
 const App = (props: IProps) => {
-  const renderCounters = () => {
-    return props.counters.map((counter: ICounter, counterId: number) => {
-      return (
-        <Counter
-          key={counterId}
-          count={counter.count}
-          decreaseCount={() => props.decreaseCount(counterId)}
-          increaseCount={() => props.increaseCount(counterId)}
-        />
-      );
-    });
+  const getKey = (item: ICounter, index: number): string => {
+    return String(index);
+  };
+
+  const renderCounter = ({
+    item,
+    index,
+  }: {
+    item: ICounter;
+    index: number;
+  }) => {
+    return (
+      <Counter
+        key={index}
+        count={item.count}
+        decreaseCount={() => props.decreaseCount(index)}
+        increaseCount={() => props.increaseCount(index)}
+        removeCounter={() => props.removeCounter(index)}
+      />
+    );
   };
 
-  return <View style={styles.container}>{renderCounters()}</View>;
+  return (
+    <View style={styles.container}>
+      <FlatList
+        contentContainerStyle={styles.container}
+        data={props.counters}
+        renderItem={renderCounter}
+        keyExtractor={getKey}
+      />
+      <Button
```

Comment:
This button is being blocked by the iOS home bar which makes tapping the button very difficult.

<img width="324" alt="Screen Shot 2019-05-16 at 12 22 49 PM" src="https://user-images.githubusercontent.com/1957315/57870606-6fdcee00-77d5-11e9-8db0-f6ef2c31f040.png">